cats/cats-core/src/main/java/com/netflix/spinnaker/cats/cache/DefaultJsonCacheData.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.cats.cache;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Clock;
+import java.util.Collection;
+import java.util.Map;
+
+public class DefaultJsonCacheData extends DefaultCacheData {
+
+  @JsonCreator
+  public DefaultJsonCacheData(
+      @JsonProperty("id") String id,
+      @JsonProperty("ttlSeconds") int ttlSeconds,
+      @JsonProperty("attributes") Map<String, Object> attributes,
+      @JsonProperty("relationships") Map<String, Collection<String>> relationships) {
+    super(id, ttlSeconds, attributes, relationships, Clock.systemDefaultZone());
+  }
+}
```

Comment:
Hmm, why not use Jackson mixins? I don't think this class is necessary.

Reply:
Right. I saw the mixins in keel, and that would probably be the long term solution. I didn't see an example of `JsonCreator` and wasn't sure how to ensure those all got applied. I was hoping to get this in before code drifted and then iterate on it further by adding the api-mixin module like keel has. There is a usage of JsonIgnore that would be more straightforward. This was just a straight forward way to avoid the Jackson dependency in the short term.

Reply:
Ok, that works for me. We can loop back.

Fwiw, I'm 99% sure we should be able to add a constructor on the mixin abstract class and annotate that with `@JsonCreator`: https://github.com/FasterXML/jackson-docs/wiki/JacksonMixInAnnotations